src/commands.cc

```diff
@@ -842,7 +842,8 @@ const CommandDesc add_hook_cmd = {
     "  * window: hook is executed only for the current window\n",
     ParameterDesc{
         { { "group", { true, "set hook group, see remove-hooks" } },
-          { "always", { false, "run hook even if hooks are disabled" } }},
+          { "always", { false, "run hook even if hooks are disabled" } },
+          { "oneshot", { false, "run the hook only once" } } },
```

Comment:
I liked `once` better, `oneshot` is less clear to me (whats a shot ?)

Reply:
The term was suggested to me by `systemd` services, whose [type](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=) can be `oneshot`.